src/client/components/ConsentsSubHeader.tsx

```diff
@@ -40,224 +48,177 @@ const h1ResponsiveText = css`
-    background-color: ${brand[400]};
-  }
+const li = (index: number, status: PageStatus) => {
+  const progressSections = CONSENTS_PAGES_COUNT - 1;
+  const position = index / progressSections;
 
-  & svg {
-    display: none;
-  }
-  &.complete svg {
+  return css`
+    ${textSans.xxsmall()}
     position: absolute;
-    display: block;
-    stroke: white;
-    fill: white;
-    height: ${CIRCLE_DIAMETER}px;
-    width: ${CIRCLE_DIAMETER}px;
-    margin: ${BORDER_SIZE}px;
-    top: 0;
-    left: 0;
-    z-index: 1;
-  }
-`;
+    left: ${position * 100}%;
+    color: ${status === 'pending' ? PENDING_COLOR : neutral[7]};
+
+    > span {
+      top: ${CIRCLE_DIAMETER + space[1]}px;
+      position: absolute;
+      width: 90px;
+      left: -45px;
+      text-align: center;
+
+      /* Hack, see explanation below */
+      white-space: nowrap;
+    }
 
-const pageProgression = css`
-  margin-top: ${space[5]}px;
-  margin-bottom: 0;
-  li {
-    color: ${neutral[60]};
-    &.active,
-    &.complete {
-      color: ${neutral[0]};
+    &:first-of-type > span {
+      left: -${CIRCLE_RADIUS}px;
+      text-align: left;
+
+      /**
+       * This allows us to force only "Create password" to wrap on mobile
+       */
+      ${until.mobileLandscape} {
+        white-space: normal;
+        width: 60px;
+      }
     }
-    &::after {
-      background-color: ${neutral[60]};
+
+    &:last-of-type > span {
+      right: -${CIRCLE_RADIUS}px;
+      left: auto;
+      text-align: right;
     }
+
     &::before {
-      border: 2px solid ${neutral[60]};
-      background-color: white;
-    }
-    &:first-child::before {
-      left: 0;
-    }
-    &:last-child::before {
-      background-color: ${neutral[60]};
+      content: '';
+      border-radius: 50%;
+      width: ${CIRCLE_DIAMETER}px;
+      height: ${CIRCLE_DIAMETER}px;
+      position: absolute;
+      left: -${CIRCLE_RADIUS}px;
+      /* TODO: apply this to psuedo elements globally */
+      box-sizing: border-box;
+      ${status === 'active' &&
+      `
+        background: ${neutral[100]};
+        border: ${COMPLETED_BORDER_SIZE}px solid ${COMPLETED_COLOR};
+      `}
+      ${status === 'pending' &&
+      `
+        background: ${neutral[100]};
+        border: ${PENDING_BORDER_SIZE}px solid ${PENDING_COLOR};
+      `}
+      ${status === 'complete' &&
+      `
+        background: ${brand[400]}
+      `}
     }
-    &:last-child::after {
-      border: 2px solid ${neutral[60]};
-      background-color: white;
-    }
-  }
-`;
 
-const ul = css`
-  display: flex;
-  list-style: none;
-  justify-content: space-between;
-  height: 54px;
-  padding: 0;
-  margin: 0;
+    ${status === 'active' && textSans.xxsmall({ fontWeight: 'bold' })}
+  `;
+};
+
+const progressWrapper = css`
+  padding: 0 ${CIRCLE_RADIUS}px;
+  overflow: hidden;
 `;
 
-const maxContentOverride = (isLastPage: boolean) => css`
-  width: ${isLastPage ? 'max-content' : 'initial'};
+const hiddenText = css`
+  display: none;
```

Comment:
just leaving as a note here – as discussed, this should probably be `visibility: hidden;` / using the Source visually hidden rules

Reply:
Have adjusted this now.